calypso/csadmin/main.go

```diff
@@ -133,9 +134,23 @@ func dkgStart(c *cli.Context) error {
 		return xerrors.New("proof for the given \"--instid\" not found")
 	}
 
+	var ltsInfo calypso.LtsInstanceInfo
+	_, ltsBuf, cid, _, err := resp.Proof.KeyValue()
+	if err != nil {
+		return fmt.Errorf("couldn't get value of Lts-instance: %v", err)
+	}
+	if cid != calypso.ContractLongTermSecretID {
+		return errors.New("not an Lts-contract")
+	}
+	err = protobuf.Decode(ltsBuf, &ltsInfo)
+	if err != nil {
+		return fmt.Errorf("couldn't decode the Lts-instance: %v", err)
+	}
+
 	reply := &calypso.CreateLTSReply{}
 	oc := onet.NewClient(cothority.Suite, calypso.ServiceName)
-	err = oc.SendProtobuf(cl.Roster.List[0], &calypso.CreateLTS{
+	log.Info("Sending CreateLTS to", ltsInfo.Roster.List[0].URL)
```

Comment:
This might break someone that uses the `--export` and expects only the public key to be printed.

Reply:
added an `if !export` in front of the log.Info. Generally speaking, I'm always at a loss to know what's happening when I use the `csadmin`, `bcadmin` commands. So I started adding more `log.Info` to them.

Do you already have a guideline for the dedis/dela, how you can get data out of the CLIs? JSON on stdout? having a `--output=file`? Just curious.

Reply:
no we don't have one yet. For me the best way to get data out of a cli is using stdout, therefore I tend to not print any log.

Reply:
true - but as a user I really like to know what the tool is up to. Specifically if it's waiting for 5-10 seconds before finishing.

Reply:
or put the logging-output to stderr and the data to stdout?

Reply:
could be a solution. At the end it should also depend on the command. In some cases its ok to print some status logs when the output is not expected to be used prgramatically, like with a wget.